db/seeding.go

```diff
@@ -0,0 +1,112 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hay-i/chronologger/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func seedTemplates(ctx context.Context, database *mongo.Database) {
+	templateCollection := database.Collection("templates")
+
+	defaultTemplates := []models.Template{
+		{
+			Title:       "Default Template #1",
+			Description: "My description 1",
+			CreatedAt:   time.Now(),
+			Default:     true,
+			Questions: []models.Question{
+				{ID: primitive.NewObjectID(), Title: "Template 1, Question 1", Description: ":)", Type: models.TextQuestion},
```

Comment:
The reason this ID is needed is because mongo will auto generate the ID for the overarching collection, but these are nested and it doesn't happen automatically.